lib/ioki/model/operator/task_list.rb

```diff
@@ -66,7 +66,7 @@ class TaskList < Base
                   type:           :string
 
         attribute :matching_rank,
-                  on:   :read,
+                  on:   [:create, :read, :update],
```

Comment:
@tom-ioki Should this have a `omit_if_nil_on: [:create, :update]`? (Our specs fail :slightly_smiling_face:).

Reply:
hm according to the structure.sql there's no `NOT NULL` constraint for `matching_rank`. Also `null` is a valid type according to the task list create and update schema 🤔 

But the UI does not allow `nil` Pretty confusing.

Let's add the `omit_if_nil_on` part. Sorry for breaking stuff.